### DISCUSSION TITLE
kubectl command - page 34

### DISCUSSION THREAD

**Discussion by oadekoya12** [nigelpoulton/TheK8sBook#19] [Q&A]
`kubectl apply -f https://raw/githubusercontent.com...` does not work I got
![image](https://user-images.githubusercontent.com/9222760/152835644-4dbd4bba-c029-45de-b994-dcbd05a1255f.png)

**Comment by ashvinmanoj** [comment]
You were supposed to copy the entire content from the previous step. The book mentions that after creating node1, the output will display 3 commands, the 2nd one should be used here. You have only written what you see in the book. But the actual syntax is much longer in length. Please check the previous steps and copy the 2nd command and execute it. Hope this helps!
